cds_ils/ldap/api.py

```diff
@@ -254,14 +252,18 @@ def import_new_ldap_users(new_ldap_users, log_func):
     )
 
 
-def delete_users(dry_run=True):
+def delete_users(dry_run=True, mark_for_deletion=False):
```

Comment:
Should it be `True` by default maybe? It looks like that the `False` value is an admin thingy?

Reply:
I left false by default since it is used in cli as well, but I can replace it there too, and have `True` by default, it will be safer indeed